src/components/privatePlaylist.vue

```diff
@@ -0,0 +1,70 @@
+<script setup lang="ts">
+// import filters from "@/utils/filters";
+import { ref, Ref } from "vue";
+import {
+  TrackCollectionApi,
+  TrackCollectionDetails,
+  Configuration,
+} from "@bcc-code/bmm-sdk-fetch";
+
+const collections: Ref<TrackCollectionDetails[]> = ref([]);
+
+new TrackCollectionApi(
+  new Configuration({
+    basePath: import.meta.env.VITE_API_URL,
+    headers: {
+      Authorization: `Bearer ${localStorage.getItem("token")}`,
+      "Accept-Language": "nb,en,zxx",
+    },
+  })
+)
+  .trackCollectionGet()
+  .then((list) => {
+    collections.value = list;
+    // eslint-disable-next-line no-console
+    console.log(list);
+  })
+  .catch(() => {});
+</script>
+
+<template>
+  <div>
+    <h3>Playlists</h3>
+    <ul v-for="collection in collections" :key="collection.id || 0">
+      <li>
+        <a href="">{{ collection.name }}</a>
```

Comment:
Would be nice to here have links to the route "PrivatePlaylist". The commit fbd733726515a29ba3a6585fccf4a6da5566693e should give you a good start :)